ConditionsConsumed/runCMSDrivers_mc2022.sh

```diff
@@ -0,0 +1,83 @@
+
+
+#6: HLT
+echo "------------------------------"
+echo "Running step-4/7: HLT"
+echo "------------------------------"
+#DIGI:pdigi_valid,L1,DIGI2RAW,HLT:@relval2022
+cmsDriver.py step6  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s HLT:@relval2022  --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step3_DIGI_L1_DIGI2RAW.root --fileout output_step6_HLT.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' --outputCommands "keep *" |& tee output_step6_HLT.log
+
+
+#6.1: DIGI_L1+DIGI2RAW+HLT
+cmsDriver.py step3  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s DIGI:pdigi_valid,L1,DIGI2RAW,HLT:@relval2022 --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step2_SIM.root --fileout output_step3_DIGI_L1_DIGI2RAW_HLT.root  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' --outputCommands 'keep *_simCastorDigis_*_*' |& tee output_step3_DIGI_L1_DIGI2RAW_HLT.log
+
+
+#7 AODSIM (RAW2DIGI,L1Reco,RECO,RECOSIM,EI)
+echo "------------------------------"
+echo "Running step-5/7: AODSIM"
+echo "------------------------------"
+cmsDriver.py step7 --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s RAW2DIGI,L1Reco,RECO,RECOSIM --datatier AODSIM --eventcontent AODSIM --filein file:output_step3_DIGI_L1_DIGI2RAW_HLT.root --fileout output_step7_AODSIM.root --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' |& tee output_step7_AODSIM.log
+
+
+#7.1 AODSIM+DIGI_L1+DIGI2RAW+HLT
+
+#8 MINIAODSIM
+echo "------------------------------"
+echo "Running step-6/7: MINIADOSIM"
+echo "------------------------------"
+cmsDriver.py step8 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s PAT --datatier MINIAODSIM --eventcontent MINIAODSIM --filein file:output_step7_AODSIM.root --fileout output_step8_MINIAODSIM.root --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' |& tee output_step8_MINIAODSIM.log
+
+
+
+#9 NANOAODSIM
+echo "------------------------------"
+echo "Running step-7/7: NANOAODSIM"
+echo "------------------------------"
+# cmsDriver.py step5 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s NANO --datatier NANOAODSIM --eventcontent NANOAODSIM --filein file:output_step7_AODSIM.root --fileout output_step9_NANOAODSIM.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' |& tee output_step9_NANOAODSIM.log
+cmsDriver.py step5 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s RAW2DIGI,L1Reco,RECO,RECOSIM,PAT,NANO --datatier NANOAODSIM --eventcontent NANOAODSIM --filein file:output_step8_MINIAODSIM.root --fileout output_step9_NANOAODSIM.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' |& tee output_step9_NANOAODSIM.log
```

Comment:
Is there a way to run the NANO step separately?

Reply:
Yes we should definitely run this step by step